docs/knowledgebase/advanced/consensus.md

```diff
@@ -31,7 +34,7 @@ different orders, and thus they must use a more elaborate method to exclude tran
 further complication, blockchain networks strive to be fault tolerant, which means that they should
 continue to provide consistent data even if some participants are not following the rules.
 
-Blockchains batch transactions into blocks and have some method to select which participant has the
+Blockchains batch transactions into blocks with methodologies to select which participant has the
```

Comment:
Why plural? Each chain has a single method to elect the next block author. And why switch to "with"? Before being modified, this sentence made two statements about blockchains. First, that they batch transactions into blocks, and second that they have a method to select... The modified version sounds to me like the "methodologies" ((probably meaning the consensus engine) somehow perform the batching.